app/views/shared/_refine_search.html.erb

```diff
@@ -1,37 +1,48 @@
 <%= form_tag studies_path, method: :get, class: 'search-form' do %>
   <div class="results-header">
-    <h3>Refine your search</h3>
     <div class="clearfix">
-      <h3 class="pull-left"><i class="fa fa-search"></i><%= text_field_tag 'search[q]', search_value(params[:search], 'q'), class: 'input-search typeahead', autocomplete: 'off' %></h3>
-      <div class="form-group pull-left healthy-volunteers">
-        <label for="search_healthy_volunteers" data-toggle='popover' data-title='Healthy Volunteer' data-content='A person who does not have the condition or disease being studied.' data-placement='top'>Healthy Volunteers <i class="fa fa-question-circle"></i></label>
-        <% healthy_voulunteers = params[:search]['healthy_volunteers'] if !params[:search].nil? and !params[:search]['healthy_volunteers'].nil? %>
-        <select id="search_heathy_volunteers" name="search[healthy_volunteers]" class="form-control">
-          <option value=''>Please Select</option>
-          <option value='1' <% if healthy_voulunteers == '1' %>selected<%end%> >Yes</option>
-          <option value='0' <% if healthy_voulunteers == '0' %>selected<%end%> >No</option>
-          <option value=''>No Preference</option>
-        </select>
-        <label class="gender-label">Sex</label>
-        <% gender = params[:search]['gender'] if !params[:search].nil? and !params[:search]['gender'].nil? %>
-        <select id="gender" name="search[gender]" class="form-control">
-          <option value=''>Any</option>
-          <option value='Female' <% if gender == 'Female' %>selected<%end%> >Female</option>
-          <option value='Male' <% if gender == 'Male' %>selected<%end%> >Male</option>
-        </select>
+      <div class="row">
+
+        <div class="col-12 col-lg-4 col-md-5">
+          <h3 class=""><i class="fa fa-search"></i><%= text_field_tag 'search[q]', search_value(params[:search], 'q'), class: 'input-search typeahead', autocomplete: 'off' %></h3>
+        </div>
+        
+        <div class="col-12 col-lg-2 col-md-3">
+          <label for="search_healthy_volunteers" data-toggle='popover' data-title='Healthy Volunteer' data-content='A person who does not have the condition or disease being studied.' data-placement='top'>Healthy Volunteers <i class="fa fa-question-circle"></i></label>
+          <% healthy_voulunteers = params[:search]['healthy_volunteers'] if !params[:search].nil? and !params[:search]['healthy_volunteers'].nil? %>
+          <select id="search_heathy_volunteers" name="search[healthy_volunteers]" class="form-control">
+            <option value=''>Please Select</option>
+            <option value='1' <% if healthy_voulunteers == '1' %>selected<%end%> >Yes</option>
+            <option value='0' <% if healthy_voulunteers == '0' %>selected<%end%> >No</option>
+            <option value=''>No Preference</option>
+          </select>
+          <label class="gender-label">Sex</label>
```

Comment:
Since we're changing search form stuff, we should add a `for` attribute on this label:

```suggestion
         <label class="gender-label" for="gender">Sex</label>
```